app/modules/entities/components/layouts/relative_entity_layout.svelte

```diff
@@ -0,0 +1,62 @@
+<script>
+  import ImagesCollage from '#components/images_collage.svelte'
+  import { loadInternalLink } from '#lib/utils'
+
+  export let uri, entitiesByUris
+
+  $: displayedEntity = entitiesByUris[uri]
+</script>
+{#if displayedEntity}
+  <li>
+    <a
+      href={displayedEntity.pathname}
+      on:click={loadInternalLink}
+      data-data={JSON.stringify(displayedEntity.image)}
+      class="entity-link"
+    >
+      {#if displayedEntity.image.url}
+        <ImagesCollage imagesUrls={[ displayedEntity.image.url ]}
```

Comment:
[not blocking]
When we know that it will be only one image, then `ImageDiv` is preferable to `ImagesCollage` (simpler markup)

Reply:
It doesnt look like `ImageDiv` can replace `ImagesCollage` without friction, hence the need of digging into it. Example with `ImageDiv`:

![2023-05-03_23-13](https://user-images.githubusercontent.com/5363918/236051368-987fe134-5fc5-4f19-8e41-fd14856e99c8.png)